product-service/src/functions/getProductsList/index.ts

```diff
@@ -0,0 +1,16 @@
+import { handlerPath } from '@libs/handler-resolver';
+
+export default {
+    handler: `${handlerPath(__dirname)}/handler.main`,
+    events: [
+        {
+            http: {
+                method: 'get',
+                path: 'product/available',
```

Comment:
Not a big deal, but I think it's better to keep up with the best practise of REST and create URLs like:

/products/{productId} - for a product
/products - for the list of products

Reply:
fixed in pr for Task 4